waspc/src/Wasp/AI/GenerateNewProject/Operation.hs

```diff
@@ -103,7 +103,7 @@ generateOperation operationType newProjectDetails entityPlans operationPlan = do
         There should be no other text in the response, just valid JSON.
 
         Additional strong guidelines for you to follow:
-         - There should typically be at least one entity listed under `entities` field in the wasp declaration.
+         - There should typically be at least one model listed under `entities` field in the wasp declaration.
```

Comment:
My intuition was, if we call it `model` then it will know with higher certainty that we are talking about Prisma models and GPT knows probably a _lot more_ than about Wasp entities. It's a confusing thing to see both mentioned, but that's the current state of our framework and it affects the GPT instructions like this.

Reply:
I think that makes sense, but so far it was doing good with `entity`, obviously it was able to make that connection. So I would probably keep it that way, if that makes sense for us in general -> let's figure taht out, and then I think GPT can hadnle both (based on how well it worked so far).

Reply:
> but so far it was doing good with entity, obviously it was able to make that connection. So I would probably keep it that way
>  let's figure taht out, and then I think GPT can hadnle both 

I'm not sure if you want to keep both or revert back to only using `entity` as naming?

I'm more for taping into the `model` name because of the reasons mentioned above.